web-components/src/components/combobox/ComboBox.ts

```diff
@@ -1557,7 +1558,6 @@ export namespace ComboBox {
       return html`
         <div
           part="combobox"
-          aria-label=${this.ariaLabelForComboBox}
```

Comment:
Why this is removed? 
Just verify if there is no impact on below PR (CX-10274: Passing List Items Count To The Parent Fix)?
https://github.com/momentum-design/momentum-ui/pull/1561

Reply:
this was extra label on the parent container, which unnecessarily added, hence removed